game/fs.hh

```diff
@@ -53,6 +54,7 @@ static inline fs::path absolute(const fs::path& p, const fs::path& base) {
 
 typedef std::vector<std::uint8_t> BinaryBuffer;
 
+nlohmann::json readJSON(fs::path const& filename); ///< Reads a JSON file into memory.
```

Comment:
sorry, but I don't think putting this function here is a good idea. this file is meant as a wrapper for filesystem and should eventually be removed. BTW pulling a dep like json in a file that is for file manipulation is pulling a build dep in many places where json is not used. I think yo ushould put it in the place where json is used or create a special jsonhelper header/cpp is really needed.

Reply:
I think you're wrong because fs.cc/hh existed before implementing filesystem and they do already have other functions in them. Whether we might eventually transition away from it and move all those functions elsewhere is a valid point but IMO we'll cross that bridge when we get to it.